src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -122,6 +122,49 @@ export default class ReimbursementRequestService {
     return createdReimbursementRequest;
   }
 
+  /**
+   * Function to reimburse a user for their expenses.
+   *
+   * @param recipientId the user who will be receiving the reimbursement
+   * @param amount the amount to be reimbursed
+   * @param submitter the person performing the reimbursement
+   * @returns the created reimbursement
+   */
+  static async reimburseUser(recipientId: number, amount: number, submitter: UserWithTeam): Promise<Reimbursement> {
+    await validateUserIsPartOfFinanceTeam(submitter);
+
+    const totalOwed = await prisma.reimbursement_Request
+      .findMany({
+        where: { recipientId, dateDeleted: null }
+      })
+      .then((userReimbursementRequests: Reimbursement_Request[]) => {
+        return userReimbursementRequests.reduce((acc: number, curr: Reimbursement_Request) => acc + curr.totalCost, 0);
+      });
+
+    const totalReimbursed = await prisma.reimbursement
+      .findMany({
+        where: { purchaserId: recipientId },
+        select: { amount: true }
+      })
+      .then((reimbursements: { amount: number }[]) =>
+        reimbursements.reduce((acc: number, curr: { amount: number }) => acc + curr.amount, 0)
+      );
+
+    if (amount > totalOwed - totalReimbursed) {
+      throw new HttpException(400, 'Reimbursement is greater than the total amount owed to the user');
+    }
+    const newReimbursement = await prisma.reimbursement.create({
+      data: {
+        purchaserId: recipientId,
+        amount,
+        dateCreated: new Date(),
+        userSubmittedId: submitter.userId
```

Comment:
same here as above, you need to specify what value you're passing in for what field